src/main/java/com/mate/hw11/Store.java

```diff
@@ -0,0 +1,131 @@
+package com.mate.hw11;
+
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.log4j.Logger;
+import org.codehaus.jackson.type.TypeReference;
+import org.codehaus.jackson.map.ObjectMapper;
+
+public class Store {
+    private static final Logger LOGGER = Logger.getLogger(Store.class);
+    private List<Fruit> storeDatabase;
```

Comment:
firstly static variables, then non-static
[Ordering of class contents](https://mate-academy.github.io/style-guides/java/java.html#s3.4.2-ordering-class-contents)